Seulement dans src3.2.6: build
Seulement dans src3.2.6/SMESH_SRC_3.2.6: aclocal.m4
Seulement dans src3.2.6/SMESH_SRC_3.2.6: autom4te.cache
Seulement dans src3.2.6/SMESH_SRC_3.2.6: configure
Seulement dans src3.2.6/SMESH_SRC_3.2.6: configure.in
Seulement dans src3.2.6/SMESH_SRC_3.2.6: salome_adm
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Actor.cxx src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Actor.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Actor.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Actor.cxx	2007-11-27 23:55:25.000000000 +0100
@@ -409,10 +409,10 @@
   myPtsLabeledDataMapper->RemoveAllInputs();
   myPtsLabeledDataMapper->Delete();
 
-  myPtsSelectVisiblePoints->UnRegisterAllOutputs();
+  //myPtsSelectVisiblePoints->UnRegisterAllOutputs();
   myPtsSelectVisiblePoints->Delete();
 
-  myPtsMaskPoints->UnRegisterAllOutputs();
+  //myPtsMaskPoints->UnRegisterAllOutputs();
   myPtsMaskPoints->Delete();
 
   myPointLabels->Delete();
@@ -425,13 +425,13 @@
   myClsLabeledDataMapper->RemoveAllInputs();
   myClsLabeledDataMapper->Delete();
 
-  myClsSelectVisiblePoints->UnRegisterAllOutputs();
+  //myClsSelectVisiblePoints->UnRegisterAllOutputs();
   myClsSelectVisiblePoints->Delete();
 
-  myClsMaskPoints->UnRegisterAllOutputs();
+  //myClsMaskPoints->UnRegisterAllOutputs();
   myClsMaskPoints->Delete();
 
-  myCellCenters->UnRegisterAllOutputs();
+  //myCellCenters->UnRegisterAllOutputs();
   myCellCenters->Delete();
 
   myCellsLabels->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/OBJECT: SMESH_Actor.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_DeviceActor.cxx src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_DeviceActor.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_DeviceActor.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_DeviceActor.cxx	2007-11-27 23:55:25.000000000 +0100
@@ -374,7 +374,7 @@
       for( vtkIdType idType = 0, *pts, npts; aConnectivity->GetNextCell( npts, pts ); idType++ )
 	aCellLocationsArray->SetValue( idType, aConnectivity->GetTraversalLocation( npts ) );
       
-      aDataSet->SetCells( aCellTypesArray, aCellLocationsArray,aConnectivity );
+      aDataSet->SetCells( aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aConnectivity );
       SetUnstructuredGrid(aDataSet);
 
       aDataSet->GetCellData()->SetScalars(aScalars);
@@ -434,7 +434,7 @@
       for( vtkIdType idType = 0, *pts, npts; aConnectivity->GetNextCell( npts, pts ); idType++ )
 	aCellLocationsArray->SetValue( idType, aConnectivity->GetTraversalLocation( npts ) );
       
-      aDataSet->SetCells( aCellTypesArray, aCellLocationsArray,aConnectivity );
+      aDataSet->SetCells( aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aConnectivity );
       SetUnstructuredGrid(aDataSet);
 
       aDataSet->GetCellData()->SetScalars(aScalars);
@@ -516,7 +516,7 @@
     for( vtkIdType idType = 0, *pts, npts; aConnectivity->GetNextCell( npts, pts ); idType++ )
       aCellLocationsArray->SetValue( idType, aConnectivity->GetTraversalLocation( npts ) );
     
-    aDataSet->SetCells( aCellTypesArray, aCellLocationsArray,aConnectivity );
+    aDataSet->SetCells( aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aConnectivity );
 
     SetUnstructuredGrid(aDataSet);
     aDataSet->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/OBJECT: SMESH_DeviceActor.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_ExtractGeometry.cxx src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_ExtractGeometry.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_ExtractGeometry.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_ExtractGeometry.cxx	2007-11-27 23:55:25.000000000 +0100
@@ -84,7 +84,7 @@
   vtkFloatingPointType multiplier;
   vtkPoints *newPts;
   vtkIdList *newCellPts;
-  vtkDataSet *input = this->GetInput();
+  vtkDataSet *input = (vtkDataSet*) this->GetInput();
   vtkPointData *pd = input->GetPointData();
   vtkCellData *cd = input->GetCellData();
   vtkUnstructuredGrid *output = this->GetOutput();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/OBJECT: SMESH_ExtractGeometry.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Object.cxx src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Object.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Object.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/OBJECT/SMESH_Object.cxx	2007-11-27 23:55:25.000000000 +0100
@@ -436,7 +436,7 @@
   for( vtkIdType idType = 0, *pts, npts; aConnectivity->GetNextCell( npts, pts ); idType++ )
     aCellLocationsArray->SetValue( idType, aConnectivity->GetTraversalLocation( npts ) );
 
-  myGrid->SetCells( aCellTypesArray, aCellLocationsArray,aConnectivity );
+  myGrid->SetCells( aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aConnectivity );
   
   aCellLocationsArray->Delete();
   aCellTypesArray->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/OBJECT: SMESH_Object.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_ClippingDlg.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_ClippingDlg.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_ClippingDlg.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_ClippingDlg.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -182,7 +182,7 @@
     myMapper->RemoveAllInputs();
     myMapper->Delete();
 
-    myPlaneSource->UnRegisterAllOutputs();
+    //myPlaneSource->UnRegisterAllOutputs();
     myPlaneSource->Delete();
   };
 
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_ClippingDlg.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_EditMeshDlg.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_EditMeshDlg.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_EditMeshDlg.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_EditMeshDlg.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -261,10 +261,10 @@
       myPtsLabeledDataMapper->RemoveAllInputs();
       myPtsLabeledDataMapper->Delete();
 
-      myPtsSelectVisiblePoints->UnRegisterAllOutputs();
+      //myPtsSelectVisiblePoints->UnRegisterAllOutputs();
       myPtsSelectVisiblePoints->Delete();
 
-      myPtsMaskPoints->UnRegisterAllOutputs();
+      //myPtsMaskPoints->UnRegisterAllOutputs();
       myPtsMaskPoints->Delete();
 
       myPointLabels->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_EditMeshDlg.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshEditPreview.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshEditPreview.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshEditPreview.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshEditPreview.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -210,7 +210,7 @@
   for( vtkIdType idType = 0, *pts, npts; aConnectivity->GetNextCell( npts, pts ); idType++ )
     aCellLocationsArray->SetValue( idType, aConnectivity->GetTraversalLocation( npts ) );
 
-  myGrid->SetCells( aCellTypesArray, aCellLocationsArray, aConnectivity );
+  myGrid->SetCells( aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray, aConnectivity );
 
   myPreviewActor->GetMapper()->Update();
 
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_MeshEditPreview.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshPatternDlg.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshPatternDlg.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshPatternDlg.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MeshPatternDlg.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -1242,7 +1242,7 @@
       aCellLocationsArray->SetValue(idType, aConnectivity->GetTraversalLocation(npts));
 
     aGrid->SetPoints(aPoints);
-    aGrid->SetCells(aCellTypesArray, aCellLocationsArray,aConnectivity);
+    aGrid->SetCells(aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aConnectivity);
 
     aConnectivity->Delete();
     aPoints->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_MeshPatternDlg.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MoveNodesDlg.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MoveNodesDlg.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MoveNodesDlg.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MoveNodesDlg.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -564,7 +564,7 @@
   aGrid->SetPoints(aPoints);
   aPoints->Delete();
 
-  aGrid->SetCells(aCellTypesArray,aCellLocationsArray,aCells);
+  aGrid->SetCells(aCellTypesArray,(vtkIdTypeArray*)aCellLocationsArray,aCells);
   aCellLocationsArray->Delete();
   aCellTypesArray->Delete();
   aCells->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_MoveNodesDlg.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MultiEditDlg.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MultiEditDlg.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MultiEditDlg.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_MultiEditDlg.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -1346,7 +1346,7 @@
     aCellLocationsArray->SetValue(idType, aConnectivity->GetTraversalLocation(npts));
 
   aGrid->SetPoints(aPoints);
-  aGrid->SetCells(aCellTypesArray, aCellLocationsArray,aConnectivity);
+  aGrid->SetCells(aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aConnectivity);
 
   // Create and display actor
   vtkDataSetMapper* aMapper = vtkDataSetMapper::New();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_MultiEditDlg.cxx~
diff -abur src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_NodesDlg.cxx src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_NodesDlg.cxx
--- src3.2.6_orig/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_NodesDlg.cxx	2007-04-24 18:41:03.000000000 +0200
+++ src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI/SMESHGUI_NodesDlg.cxx	2007-11-27 23:55:26.000000000 +0100
@@ -158,10 +158,10 @@
       vtkIdType npts;
       aCellLocationsArray->SetValue(0, aCells->GetTraversalLocation(npts));
 
-      aGrid->SetCells(aCellTypesArray, aCellLocationsArray, aCells);
+      aGrid->SetCells(aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray, aCells);
 
       aGrid->SetPoints(myPoints);
-      aGrid->SetCells(aCellTypesArray, aCellLocationsArray,aCells);
+      aGrid->SetCells(aCellTypesArray, (vtkIdTypeArray*)aCellLocationsArray,aCells);
       aCellLocationsArray->Delete();
       aCellTypesArray->Delete();
       aCells->Delete();
Seulement dans src3.2.6/SMESH_SRC_3.2.6/src/SMESHGUI: SMESHGUI_NodesDlg.cxx~
